pyiron_atomistics/vasp/base.py

```diff
@@ -2051,15 +2073,33 @@ def collect(self, directory=os.getcwd(), sorted_indices=None):
             log_dict["forces"] = self.vp_new.vasprun_dict["forces"]
             log_dict["cells"] = self.vp_new.vasprun_dict["cells"]
             log_dict["volume"] = np.linalg.det(self.vp_new.vasprun_dict["cells"])
-            # log_dict["total_energies"] = self.vp_new.vasprun_dict["total_energies"]
-            log_dict["energy_tot"] = self.vp_new.vasprun_dict["total_energies"]
+            # The vasprun parser also returns the energies printed again after the final SCF cycle under the key
+            # "total_energies", but due to a bug in the VASP output, the energies reported there are wrong in Vasp 5.*;
+            # instead use the last energy from the scf cycle energies
+            # BUG link: https://ww.vasp.at/forum/viewtopic.php?p=19242
+            try:
+                # bug report is not specific to which Vasp5 versions are affected; be safe and workaround for all of
+                # them
+                is_vasp5 = self.vp_new.vasprun_dict["generator"]["version"].startswith(
+                    "5."
+                )
+            except KeyError:  # in case the parser didn't read the version info
+                is_vasp5 = True
+            if is_vasp5:
+                log_dict["energy_pot"] = np.array(
+                    [e[-1] for e in self.vp_new.vasprun_dict["scf_fr_energies"]]
+                )
+            else:
+                # total energies refers here to the total energy of the electronic system, not the total system of
+                # electrons plus (potentially) moving ions; hence this is the energy_pot
+                log_dict["energy_pot"] = self.vp_new.vasprun_dict["total_energies"]
             if "kinetic_energies" in self.vp_new.vasprun_dict.keys():
-                log_dict["energy_pot"] = (
-                    log_dict["energy_tot"]
-                    - self.vp_new.vasprun_dict["kinetic_energies"]
+                log_dict["energy_tot"] = (
+                    log_dict["energy_pot"]
+                    + self.vp_new.vasprun_dict["kinetic_energies"]
                 )
             else:
```

Comment:
This will never be true, because the if condition only checks whether kinetic_energies key exists or not and not if there are actual values in it. This becomes a problem for static calculations where self.vp_new.vasprun_dict["kinetic_energies"] is an empty list but the key is still there. This will result in log_dict["energy_tot"] being empty.

Reply:
Correct!  I pushed a fix.

Reply:
parsing energy now works perfectly!